staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/validation_test.go

```diff
@@ -1842,6 +1845,88 @@ func TestValidationExpressions(t *testing.T) {
 				"authorizer.path('/healthz').check('get').allowed()": "undeclared reference to 'authorizer'",
 			},
 		},
+		{name: "optionals", // https://github.com/google/cel-spec/wiki/proposal-246
```

Comment:
Should we also add optionals test to cost stability?

Reply:
Good catch. Added. Note that `optMap` is a macro.